contractcourt/channel_arbitrator.go

```diff
@@ -1996,10 +2027,62 @@ func (c *ChannelArbitrator) prepContractResolutions(
 	commitHash := contractResolutions.CommitHash
 	failureMsg := &lnwire.FailPermanentChannelFailure{}
 
+	var htlcResolvers []ContractResolver
```

Comment:
The order doesn't matter here right? Looks like we used to put anchor resolver at the end.

Reply:
Order doesn't matter, caller just launches them